packages/runtime/runtime-definitions/src/protocol.ts

```diff
@@ -65,3 +69,27 @@ export interface IAttachMessage {
 export type InboundAttachMessage = Omit<IAttachMessage, "snapshot"> & {
 	snapshot: IAttachMessage["snapshot"] | null;
 };
+
+/**
+ * This is the message type that is used within the runtime when processing a sequenced message.
+ * It is the same as ISequencedDocumentMessage, but without the contents and clientSequenceNumbers
+ * which are sent separately. The contents are modified at multiple layers in the stack so having it
+ * separate doesn't require packing and unpacking the entire message.
+ * @alpha
+ * @legacy
+ */
+export type ISequencedRuntimeMessageCore = Omit<
```

Comment:
Maybe ISequencedRuntimeMessageEnvelope? "Core" does not tell much.
Maybe "envelope" could be interpreted differently (as same word is used with slightly different context in other layers), but I think it's better than "core"

Reply:
Changed it.